config/routes.rb

```diff
@@ -1,9 +1,10 @@
 Rails.application.routes.draw do
+  resources :events
+  resources :calendars
   resources :blogposts
   root 'static_pages#home'
 
   get 'about',        to: 'static_pages#about'
-  get 'calendar',     to: 'static_pages#calendar'
```

Comment:
I deleted this. The resources on lines 2 & 3 were autogenerated when I scaffolded all the things